content/app/design/guidelines/components/select/_index.en.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: Nedtrekksliste
-description: Nedtrekksliste brukes i tilfeller der brukeren skal velge et alternativ blant flere. Det samme gjelder radioknapper, så vær bevisst på når du bruker hvilken.  
+title: Dropdown
+description: The pull-down list should be used in cases where the user is picking one of many alternatives, much like the radio buttons. 
 weight: 70
 ---
 
-Nedtrekksliste velges når:
-- Du har mange alternativer (mer enn 5)
-- Du har et anbefalt valg som vises som forhåndsvalgt
-- Det ikke er så viktig for brukeren å kunne sammlenligne alternativene
-- Du ønsker ikke at brukeren skal trenge å lese alle alternativene
-- Valgalternativene er kjent for brukeren
+### The pull-down list should be used when:
+- you have many alternatives (more than 5, but not unlimited)
+- You have a recommended choice which is pre-selected
+``
+- it isn't important for the user to compare the alternatives
+- you don't need the user to read every alternative
+- the alternatives are known by the user (birth month, country of origin, etc)
```

Comment:
Capitalizing the rest of the list as well (I only wrote a suggestion about this earlier, but never provided one)
```suggestion
- You have many alternatives (more than 5, but not unlimited)
- You have a recommended choice which is pre-selected
- It isn't important for the user to compare the alternatives
- You don't need the user to read every alternative
- The alternatives are known by the user (birth month, country of origin, etc)
```